persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java

```diff
@@ -145,6 +149,10 @@ public ProtocolVersion protocol() {
       return protocolVersion.get();
    }
 
+   /**
+    * @deprecated since 12.0 - This option can still be needed when retrieving from a preexisting remote cache
+    */
+   @Deprecated
    public boolean rawValues() {
```

Comment:
Could you please check if https://github.com/infinispan/infinispan/blob/8aa30677362ea1befcb5b9a828008dd0bc69d857/documentation/src/main/asciidoc/topics/ref_rolling_upgrade_remote_store.adoc can be updated to remove those deprecated attributes?

Reply:
TBH, I need to think about it more. I would rather leave them alone for this PR as I am not certain yet. TBH we may not be able to remove raw values even, it needs more thought. I created https://issues.redhat.com/browse/ISPN-12209 to do this.